src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -43,7 +49,7 @@
      */
     @Test
     public void logsMessage() throws IOException {
-        new TkSlf4j(new TkText("test")).act(new RqFake());
+        new TkSlf4j(new TkEmpty()).act(new RqFake());
```

Comment:
@essobedo to decrease the data coupling, `TkEmpty` is already used in another test and for this purpose, I think `TkEmpty` serves the same function as `TkText`.